css/index.css

```diff
@@ -387,7 +501,64 @@ textarea:focus {
   width: 32%;
 }
 
-@media (width >= 768px) {
+.mobile-menu-navbar {
+  display: none;
+}
+
+.mobile-navbar {
+  font-size: 45px;
+}
+
+.d-none {
+  display: none;
+}
+
+.d-block {
+  display: flex;
+  opacity: 1;
+  left: 0;
+}
+
+.d-flex {
+  display: flex;
+}
+
+@media (width >=1000px) {
+  .menu-link {
+    display: none;
+  }
```

Comment:
Can we use `d-none-md` here instead? This way we are able to define display none for all medium screens. We can do similar for very large screens by defining `d-none-xl` or `d-none-lg`.